rust/stub/src/unified_sections.rs

```diff
@@ -0,0 +1,42 @@
+/// List of PE sections that have a special meaning with respect to
+/// UKI specification.
+/// This is the canonical order in which they are measured into TPM
+/// PCR 11.
+/// !!! DO NOT REORDER !!!
```

Comment:
Perhaps adding `#[repr(u8)]` and a unit test that checks their order would be nice?

[`std::mem::discriminant`](https://doc.rust-lang.org/stable/std/mem/fn.discriminant.html) could also be used, perhaps

Reply:
Hmm, I'm not so sure how to do it, I will go ahead and merge this, can you open a PR to propose something?

Reply:
Added a suggestion :)

Reply:
Applied :)